tests/collecting/CollectorsTest.py

```diff
@@ -239,21 +239,21 @@ def test_bear_invalid(self):
     def test_simple_single(self):
         self.assertEqual(len(collect_bears(
             [os.path.join(self.collectors_test_dir, 'bears')],
-            ['bear1'],
+            ['ABear'],
```

Comment:
why is this changing?

That implies your algorithm is operating differently than before.

i.e. a regression.

Reply:
For this test case the bear file was named as `bear1`. Now when collecting the bear it would change this to `bear1Bear` which is not what we want. So, I renamed the test case bear file to `ABear`. 
The algorithm should work for all previous cases as before:
- Input `JSStandardBear` this has the `Bear` suffix at the end so does not get changed.
- Input `JSStandard` this does not have the `Bear suffix so changes to `JSStandardBear` when collecting.
-Input `JSStandardbear` changes to `JSStandardBear` when collecting

Reply:
no that is covering cases which are in our bears repos, but isnt covering cases that were previously allowed by custom bears created and used by other people, possibly confidential and not published anywhere.